stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/EventHandlerImpl.kt

```diff
@@ -431,15 +431,12 @@ internal class EventHandlerImpl(
                     domainImpl.setOffline()
                 }
                 is ConnectedEvent -> {
-                    val recovered = domainImpl.isInitialized()
-
                     domainImpl.setOnline()
                     domainImpl.setInitialized()
-                    if (recovered && domainImpl.recoveryEnabled) {
-                        domainImpl.connectionRecovered(true)
-                    } else {
-                        domainImpl.connectionRecovered(false)
-                    }
+                    domainImpl.connectionRecovered(domainImpl.recoveryEnabled)
```

Comment:
While we're here, looks like `recoveryEnabled` could be a `val`, because we never update it, we just set it at the creation of `ChatDomainImpl`.  (Also, `ChatDomainImpl` has an unused internal ctor?)

Reply:
:+1: